renderdoc/driver/gl/gl_hooks.cpp

```diff
@@ -177,6 +177,22 @@ void GLDispatchTable::PopulateWithCallback(PlatformGetProcAddr lookupFunc)
   }
 
   ForEachSupported(HookFunc);
+  GL.glBeginPerfQueryINTEL = (PFNGLBEGINPERFQUERYINTELPROC)lookupFunc("glBeginPerfQueryINTEL");
```

Comment:
So you would like to have those intel entry points added to the supported ones?
I'm having a hard time getting this to work...
If I add the chunks to gl_common.h and the same case as glGet* in gl_driver.cpp that leads to asserts.
Returning true to skip them leads to other chunks being messed up somehow.

Reply:
Can you give a bit more detail about what went wrong? Which asserts did you hit? One problem I can see is that you didn't add the intel functions at the end of the `GLChunk` enum, which would lead to binary incompatibility. If you loaded a capture made prior to your changes then I would expect that to break. Any `ContextConfiguration` chunk would instead hit the case for `glBeginPerfQueryINTEL` because that's what the enum value corresponds to now, and after that point the serialisation would be desynced and everything would blow up (hopefully it wouldn't crash but there's not necessarily a guarantee in all cases).

If you captured and replayed on a build with the changes then I don't see anything in principle that should break, it looks fine to me.

Reply:
Ah that might be the problem then. Is there a version number I should bump so that the user is warned that traces are incompatible?

Reply:
Thanks, got it.